src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSSubtaskInformation.cs

```diff
@@ -84,7 +112,7 @@ public PSComputeNodeInformation ComputeNodeInformation
             }
         }
         
-        public Microsoft.Azure.Batch.Common.TaskState? PreviousState
+        public Microsoft.Azure.Batch.Common.SubtaskState? PreviousState
```

Comment:
Does this type have all the properties of the previous type?

Reply:
In this case no, it does not, it's missing one (although in practice that 1 could never be returned, but that's a runtime thing, which I assume means that this is a breaking change)

I will also document this in upcoming-breaking-changes